cluster/aws/config-default.sh

```diff
@@ -74,7 +74,8 @@ MASTER_TAG="${INSTANCE_PREFIX}-master"
 NODE_TAG="${INSTANCE_PREFIX}-minion"
 NODE_SCOPES=""
 POLL_SLEEP_INTERVAL=3
-SERVICE_CLUSTER_IP_RANGE="10.0.0.0/16"  # formerly PORTAL_NET
+INTERNAL_CIDR="${INTERNAL_CIDR:-10.0.0.0/8}"
```

Comment:
INTERNAL is not sufficiently descriptive.  How about NON_MASQUERADE_CIDR or something?  Or maybe INTERNAL_NET_CIDR is sufficient?  I want it to be understandable without trawling code.

Add a comment like "traffic to IPs outside this range with use IP masquerade".